StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/ECE/ECEViewControllerSnapshotTests.swift

```diff
@@ -1,169 +0,0 @@
-//
-//  ECEViewControllerSnapshotTests.swift
-//  StripePaymentSheetTests
```

Comment:
These tests didn't seem useful since they only rendered a blank screen. We also don't own the screen that it is rendering so it will be difficult to maintain these. Removing these for now.

Reply:
Sounds good, I was thinking we'd have the Shop Pay logo rendered on our end, but it makes sense to just have it in rendered by the page